### DISCUSSION TITLE
diplat gitbook public shared audience links in the messages from @guitbook app instead of gitbook space links

### DISCUSSION THREAD

**Discussion by rocchisanijl** [GitbookIO/community#397] [Feature Requests]
I just test the integration with slack app
It is a great feature but something should be modified or at least customize
When I request a question via @gitbook app I got links to my space gitbook app.gitbook.com/...
but I would like to have shared links for the public audience 
because my chatters are not all members of my organization

I would be delighted if you can add this feature :smile:

**Comment by Flohs** [comment]
We would require it as well, due to the same reasons. Customers should be able to ask via Slack and when they receive a link, they should get the link of the custom domain that was set up in gitbook. Same goes for snippets. Unfortunatelly snippets can't be public, so just removing the "source" from the slack response, when the answer cam from a snippet, might help already.

**Reply by ThorstenPlock** [reply]
We also need this functionality for our customers' convenience. It should allow them to inquire via Slack, and upon receiving a link, they should be directed to the custom domain set up in GitBook. Similarly, this should apply to snippets. Unfortunately, snippets can't be made public, so simply omitting the 'source' from the Slack response when the answer comes from a snippet could be a helpful workaround.

**Reply by plockbrothers** [reply]
We need this feature for the same reasons. Customers should be able to ask via Slack and receive a link pointing to the custom domain set up in GitBook. The same applies to snippets. However, snippets cannot be made public, so removing the 'source' from the Slack response when the answer comes from a snippet might suffice.

**Comment by plockbrothers** [comment]
We need this feature for the same reasons. Customers should be able to ask via Slack and receive a link pointing to the custom domain set up in GitBook. The same applies to snippets. However, snippets cannot be made public, so removing the 'source' from the Slack response when the answer comes from a snippet might suffice.

**Comment by ThorstenPlock** [comment]
We also need this functionality for our customers' convenience. It should allow them to inquire via Slack, and upon receiving a link, they should be directed to the custom domain set up in GitBook. Similarly, this should apply to snippets. Unfortunately, snippets can't be made public, so simply omitting the 'source' from the Slack response when the answer comes from a snippet could be a helpful workaround.
